src/pages/Graph/GraphToolbar/GraphFind.tsx

```diff
@@ -79,34 +82,41 @@ export class GraphFind extends React.PureComponent<GraphFindProps, GraphFindStat
     }
   }
 
+  // We only update on a change to the find/hide/compress values, or a graph change.  Although we use other props
+  // in processing (compressOnHide, layout, etc), a change to those settings will generate a graph change, so we
+  // wait for the graph change to do the update.
+  shouldComponentUpdate(nextProps: GraphFindProps) {
+    const cyChanged = this.props.cy !== nextProps.cy;
+    const findChanged = this.props.findValue !== nextProps.findValue;
+    const hideChanged = this.props.hideValue !== nextProps.hideValue;
+    const graphChanged = this.props.updateTime !== nextProps.updateTime;
+
+    return cyChanged || findChanged || hideChanged || graphChanged;
```

Comment:
So, this ignores `nextProps.showFindHelp`. This means that when you click the "Find/Hide help" button, this component doesn't re-renders and the help dialog is blocked until the next graph update happens and, suddenly, the help dialog makes its long awaited appearance. I think you need take into account that property, hoping that won't break all this good work.

Reply:
@israel-hdez Another great catch, you could have a second job in QE :)  Fortunately, this is purely local to graphFind and I've added it back.  Should all be good now.  I'm going to release this to QE.  Thanks for all of your review.